### PR TITLE
New package: GaijinNetwork.WarThunder.CDK version 2024.07.24

### LINKED ISSUE #189983: [Package Request]: `GaijinNetwork.WarThunder.CDK`
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [x] The installer meets the above requirements

### Please provide the following information

<table><tr>
	<th>Download Page URI</th>
	<td><a href="https://wiki.warthunder.com/index.php?title=Download_War_Thunder_CDK&oldid=190725"><code>wiki.warthunder.com/index.php?title=Download_War_Thunder_CDK</code></a>
</tr><tr>
	<th>Publisher</th>
	<td><a href="https://github.com/microsoft/winget-pkgs/tree/f6c284c792e4d01f04dcd3b2520ed09b7fd8715c/manifests/g/GaijinNetwork/"><code>/manifests/g/GaijinNetwork/</code></a></td>
</tr><tr>
	<th>Package Name</th>
	<td><code>GaijinNetwork.WarThunder.CDK</code></td>
</tr><tr>
	<th>Description</th>
	<td><blockQuote cite="https://wiki.warthunder.com/index.php?title=War_Thunder_CDK&oldid=123684#:~:text=War%20Thunder%20CDK%20(Content%20Development%20Kit)%20is%20a%20set%20of%20tools%20for%20user%20content%20creation%20available%20to%20every%20player.">War Thunder CDK (Content Development Kit) is a set of tools for user content creation available to every player. </blockQuote></td>
</tr><tr>
	<th>Package Version</th>
	<td><code>2024_07_24__17_35</code></td>
</tr><tr>
	<th>Installer URI</th>
	<td><a href="https://cdk.warthunder.com/WarThunderCDK_2024_07_24__17_35.exe"><code>https://cdk.warthunder.com/WarThunderCDK_2024_07_24__17_35.exe</code></a></td>
</tr></table>

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #189983
- [x] Waiting for #190448
  - Which will be resolved by #190449 

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Additional information

It seems this package depends on **HUGE** (from `19.6GB` minimum, to `115.8GB` top quality) game contents of _War Thunder_.

Game contents are too large for me. Appreciate if other contributors & volunteers can investigate it further. 🙏🏼

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/190452)

### CODE REVIEW

**Comment by Dragon1573** [comment]
> The package manager bot determined there was an issue with installing the application correctly. Please check the application installs correctly. Once repaired, please push an update to your pull request.
> 
> Template: msftbot/validationError/installation/general

This problem is already mentioned in #190448. It will be resolved by #190449.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978334960
> 2024-11-11 07:59:11.270 [CLI ] Package hash verification failed. SHA256 in manifest [58793f14b277bd13935e37a1b4c7771b341e96579fe5e164b351db498225ed2d] does not match download [60ce5ea054da2b0e952d3cb0be5297e13bc9a0954fb8e19e64b42a7d6b46c007]

Automated error analysis suggests -1978334960 may mean Failed to install package dependencies.

(Automated response - build 897.)

**Comment by Dragon1573** [comment]
- See https://github.com/microsoft/winget-pkgs/pull/190452#issuecomment-2470021190

/AzurePipelines run

File: manifests/g/GaijinNetwork/WarThunder/CDK/2024.07.24/GaijinNetwork.WarThunder.CDK.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+UpgradeBehavior: install
+ProductCode: '{ed8deea4-29fe-1932-9612-e2122d8a62d9}}_is1'
+ReleaseDate: 2024-07-24
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: GaijinNetwork.WarThunder
+Installers:
+- Architecture: x64
+  InstallerUrl: https://cdk.warthunder.com/WarThunderCDK_2024_07_24__17_35.exe
+  InstallerSha256: 1B0AD20DCDA5060646FDE17779E91EBCC256CEAD5FF80BBEA1CF90F90F9FA3DF
```
Comment (stephengillie) [inline manifests/g/GaijinNetwork/WarThunder/CDK/2024.07.24/GaijinNetwork.WarThunder.CDK.installer.yaml:21]:
```suggestion
  InstallerSha256: 60CE5EA054DA2B0E952D3CB0BE5297E13BC9A0954FB8E19E64B42A7D6B46C007
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> Automatic Validation ended with:
> 
> > Installation failed with exit code -1978334960
> > 2024-11-11 07:59:11.270 [CLI ] Package hash verification failed. SHA256 in manifest [58793f14b277bd13935e37a1b4c7771b341e96579fe5e164b351db498225ed2d] does not match download [60ce5ea054da2b0e952d3cb0be5297e13bc9a0954fb8e19e64b42a7d6b46c007]
> 
> Automated error analysis suggests -1978334960 may mean Failed to install package dependencies.
> 
> (Automated response - build 897.)

**Review by ItzLevvie — APPROVED** [review]

**Comment by Dragon1573** [comment]
> [!CAUTION]
>
> It seems the installer file changes frequently. We're unable to pass the validation pipeline. 😞

**Pull request closed without merging**